### PR TITLE
Update on the README.md

### PR DESCRIPTION
**Pull request by Enabuufu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
-
-` git clone https://github.com/toolforge/paws.git `
-
-`cd paws`
+[PAWS](https://www.mediawiki.org/wiki/PAWS): A Web Shell (PAWS) is a Jupyter notebooks deployment that has been customized to make interacting with Wikimedia wikis easier. It allows users to create and share documents that contain live code, visualizations such as graphs, rich text, etc. The user created notebooks are a powerful tool that enables data analysis and scientific research, and also transforms the way in which programmers write code - by enabling an exploratory environment with a quick feedback loop, and a low barrier for entry through it's easy to use graphical interface. 
+
+It is a customized JupyterHub on Kubernetes running inside Wikimedia Cloud Services. It provides a simple way to use Python and pywikibot interact with the Wikimedia Projects, including the various wikis via OAuth, the Wiki Replicas Cloud service, and others. 
+
+# How to contribute
+1. Just simply clone on your computer by writing the following command: git clone https://github.com/toolforge/paws.git
```
Comment (aborrero) [inline README.md:8]: Leaving only the command here should be fine.

File: README.md [hunk 2]
```diff
@@ -37,6 +46,14 @@ A notebook can be turned into a public notebook by publishing a link to it. This
 If you want to run the copy yourself, or do interactive changes, you must download the notebook and reupload on your own account. Downloading the raw format of the previous example can be done by adding format=raw to the previous example …revisions-sql.ipynb?format=raw. This download-reupload-process is somewhat awkward.
 
 Note that a notebook will always be published, as the link can be guessed, so don't add any private information. 
+# chat
+ Applicants can get help and feedback from both mentors and community members. Community members discuss their contributions in a public chat.
```
Comment (aborrero) [inline README.md:50]: We hope not only GSoC/Outreachy applicants contribute to this project. You should rewrite this sentence as if we were inviting anyone to contribute.

Example:

"Contributors can get help and feedback from other community members in a public IRC channel. [...]"

File: README.md [hunk 3]
```diff
@@ -37,6 +46,14 @@ A notebook can be turned into a public notebook by publishing a link to it. This
...
+Please introduce yourself on the public project chat, follow the link  [IRC chat](https://webchat.freenode.net/#wikimedia-cloud)
+
+# Try paws yourself
+Try out paws yourself inorder to familiarise your self with it, all you need is to login with your wikimedia account and you're good to go. Follow the link [create a note book](https://paws.wmflabs.org/paws/hub/login)
```
Comment (aborrero) [inline README.md:54]: Typos 'inorder', 'your self'.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
-
-How to contribute? Just simply clone on your computer by writing the following command: 
-
-` git clone https://github.com/toolforge/paws.git `
-
-`cd paws`
+[PAWS](https://www.mediawiki.org/wiki/PAWS): A Web Shell (PAWS) is a Jupyter notebooks deployment that has been customized to make interacting with Wikimedia wikis easier. It allows users to create and share documents that contain live code, visualizations such as graphs, rich text, etc. The user created notebooks are a powerful tool that enables data analysis and scientific research, and also transforms the way in which programmers write code - by enabling an exploratory environment with a quick feedback loop, and a low barrier for entry through it's easy to use graphical interface. 
+
+It is a customized JupyterHub on Kubernetes running inside Wikimedia Cloud Services. It provides a simple way to use Python and pywikibot interact with the Wikimedia Projects, including the various wikis via OAuth, the Wiki Replicas Cloud service, and others.
```
Comment (aborrero) [inline README.md:5]: Please reword this sentence to make more sense.

Specifically: 'It provides a simple way to use Python and pywikibot to interact with the wikimedia projects [..]" (note missing 'to'.)

**Comment by Enabuufu** [comment]
OK, thanks for the comments,

Another PR is submitted in response to the comments, please review it

**Comment by crookedstorm** [comment]
Closing from age and conflicts

**Pull request closed without merging**